packages/connector-ftp/.storybook/StoryHelper.css

```diff
@@ -0,0 +1,19 @@
+@import url('~patternfly/dist/css/patternfly.css');
+@import url('~patternfly/dist/css/patternfly-additions.css');
+@import url('~patternfly-react/dist/css/patternfly-react.css');
+
+html,
+body,
+#root {
+  height: 100%;
+}
+
+#root {
```

Comment:
Not too familiar with Storybook or this setup, but is it necessary to use an ID over a class here?